Tweets/_posts/Bear market, bullish development/2021-07-22-Bear-market,-bullish-development.md

```diff
@@ -1,58 +1,63 @@
 ---
 layout: post
-title:  "Bear market, bullish development"
+title:  "Bear market, dévelopement bullish"
 categories: [ Tweets ]
 image: ./Bear-market,-bullish-development/image1.jfif
 author: Dudesahn
 translator:
```

Comment:
`translator: Cryptoouf`
add your name or whichever name you want to go by here has to be the same everywhere